### PR TITLE
Updated Tank3

### PR DESCRIPTION
**Pull request by Dzierzan**
Seperated turret from tank3's chassis to make it an actual turret. I also remapped the actor.

### CODE REVIEW

**Comment by GraionDilach** [comment]
Shouldn't the turret have a black edge to complement the other units?

**Comment by Dzierzan** [comment]
At first I wanted to draw black outline around the turret but I wasn't 100% certain of that. I kinda wanted to preserve a bit of original turret's art.

File: mods/hv/mod.yaml [hunk 1]
```diff
@@ -1,229 +1,229 @@
...
-	fastest:
-		Name: Fastest
-		Timestep: 20
-		OrderLatency: 6
-
-ColorValidator:
-	TeamColorPresets: d0a090, 5391d3, 7e4865, 2f9b1c
+Metadata:
+	Title: OpenHV
+	Version: f0c808d2fc2a5df94660c1d32efaa5f96c4b3626
```
Comment (Mailaender) [inline mods/hv/mod.yaml:3]: You accidentally typed `make version` please undo.

File: mods/hv/rules/ships.yaml [hunk 2]
```diff
@@ -30,7 +30,7 @@ BOAT:
 		Recoil: 75
 		RecoilRecovery: 25
 		MuzzleSequence: muzzle
-		LocalOffset: 200,0,-50
+		LocalOffset: 0,0,0
```
Comment (Mailaender) [inline mods/hv/rules/ships.yaml:33]: Is this on purpose?

File: mods/hv/rules/vehicles.yaml [hunk 3]
```diff
@@ -82,13 +82,20 @@ TANK3:
 	RevealsShroud:
 		Range: 6c0
 	Turreted:
-		TurnSpeed: 7
+		TurnSpeed: 5
+		Offset: -125,0,50
 	Armament:
-		Weapon: 30mm
```
Comment (Mailaender) [inline mods/hv/rules/vehicles.yaml:87]: This weapon is now unused. Is there a reason you removed and duplicated it with `TyrianTankCannon`?

File: mods/hv/rules/ships.yaml (hunk 2 above)
Comment (Dzierzan) [reply mods/hv/rules/ships.yaml:33]: A mistake. I was in ships.yaml and I must have done a change which I accidentally commited.

File: mods/hv/rules/vehicles.yaml (hunk 3 above)
Comment (Dzierzan) [reply mods/hv/rules/vehicles.yaml:87]: IMO weapons have to totally refactored. I can revert it to 30mm. Although I'd love tank3 and tyrian tank to be just reskins (mirrored units) for two factions.

Comment (GraionDilach) [reply mods/hv/rules/vehicles.yaml:87]: Not just weapons. At this point this project is still just adding things. There should be a design document at one point to formulate the gameplay onwards with techtrees and whatnot.

File: mods/hv/mod.yaml [hunk 4]
```diff
@@ -1,229 +1,229 @@
-Metadata:
-	Title: OpenHV
```
Comment (Mailaender) [inline mods/hv/mod.yaml:2]: You still changed all the line endings in here.

**Comment by Mailaender** [comment]
Can you https://support.gitkraken.com/working-with-commits/squash/ the fixup commits?

**Pull request closed without merging**